docs/managing_deis/backing_up_data.rst

```diff
@@ -195,6 +195,12 @@ Note that you'll need to specify the SSH port when using Vagrant:
 Finishing up
 ~~~~~~~~~~~~
 
+.. note::
+
+    As of version 1.11.0, the following steps are no longer necessary because
+    the ``deis-controller`` component will handle them automatically upon
+    starting.
```

Comment:
I agree. Either that or a TODO to delete this note once v1.11.0 is released since "latest" is what people are usually looking at.